remotespark/wrapperkernel/sparkkernelbase.py

```diff
@@ -54,59 +54,64 @@ def __init__(self, implementation, implementation_version, language, language_ve
 
     def do_execute(self, code, silent, store_history=True, user_expressions=None, allow_stdin=False):
         if self._fatal_error is not None:
-            self._repeat_fatal_error()
+            return self._repeat_fatal_error()
+        try:
+            return self._do_execute(code, silent, store_history, user_expressions, allow_stdin)
+        except Exception as e:
```

Comment:
I am unsure if that's true.  Can you give me an example of something that might trigger the error.